src/components/Book.js

```diff
@@ -0,0 +1,19 @@
+import PropTypes from 'prop-types';
+
+const Book = ({ book }) => {
+  const { bookId, bookTitle, bookCategory } = book;
+  return (
+    <tr>
+      <td>{bookId}</td>
+      <td>{bookTitle}</td>
+      <td>{bookCategory}</td>
+    </tr>
+  );
+};
+
+Book.propTypes = {
+  // eslint-disable-next-line react/forbid-prop-types
```

Comment:
## Linter Disabled ⚠️ 
- [ ] There is an easy fix for this rule instead of the disable option. Let's lift this and improve on our propTypes definition, thank you!

Reply:
Fixed this as well